frontend/src/pages/org/dashboard.ts

```diff
@@ -0,0 +1,260 @@
+          ${this.renderCard(
+            msg("Storage"),
+            (metrics) => html`
+              <div class="font-semibold mb-3">
+                <sl-format-bytes
+                  value=${metrics.storageUsedBytes ?? 0}
+                ></sl-format-bytes>
+                ${msg("Used")}
+              </div>
+              <dl>
+                ${this.renderStat({
+                  value: metrics.archivedItemCount,
+                  singleLabel: msg("Archived Item"),
+                  pluralLabel: msg("Archived Items"),
+                  icon: "file-zip-fill",
+                })}
+                ${this.renderStat({
+                  value: metrics.crawlCount,
+                  singleLabel: msg("Crawl"),
+                  pluralLabel: msg("Crawls"),
+                  icon: "gear-wide-connected",
+                })}
+                ${this.renderStat({
+                  value: metrics.uploadCount,
+                  singleLabel: msg("Upload"),
+                  pluralLabel: msg("Uploads"),
+                  icon: "upload",
+                })}
+                ${this.renderStat({
+                  value: metrics.profileCount,
+                  singleLabel: msg("Browser Profile"),
+                  pluralLabel: msg("Browser Profiles"),
+                  icon: "window-fullscreen",
+                })}
+              </dl>
+            `,
+            (metrics) => html`<footer class="mt-4 flex justify-end">
+              <sl-dropdown
+                distance="4"
+                placement="bottom-end"
+                @sl-select=${(e: SlSelectEvent) => {
+                  this.dispatchEvent(
+                    <SelectNewDialogEvent>new CustomEvent("select-new-dialog", {
+                      detail: e.detail.item.value,
+                    })
+                  );
+                }}
+              >
+                <sl-button
+                  slot="trigger"
+                  size="small"
+                  caret
+                  ?disabled=${metrics.storageQuotaBytes > 0 &&
+                  metrics.storageUsedBytes >= metrics.storageQuotaBytes}
+                >
+                  <sl-icon slot="prefix" name="plus-lg"></sl-icon>
+                  ${msg("Add New...")}
+                </sl-button>
+                <sl-menu>
+                  <sl-menu-item value="browser-profile">
+                    ${msg("Browser Profile")}
+                  </sl-menu-item>
+                  <sl-menu-item value="upload">${msg("Upload")}</sl-menu-item>
+                </sl-menu>
+              </sl-dropdown>
+            </footer> `
+          )}
+          ${this.renderCard(
+            msg("Crawling"),
+            (metrics) => html`
+              <dl>
+                ${this.renderStat({
+                  value: metrics.workflowsRunningCount,
+                  singleLabel: msg("Crawl Running"),
+                  pluralLabel: msg("Crawls Running"),
+                  icon: "record-fill",
```

Comment:
Crawling indicators use `dot` from the `app` library.  Guess this would need the library to be specified at all times?

Also obviously these are in colour in the mockups, so I assume you were planning on updating this along with the graph implementation?